evm_trace/vmtrace.py

```diff
@@ -5,7 +5,7 @@
 from eth.vm.memory import Memory
 from eth.vm.stack import Stack
 from eth_utils import to_checksum_address
-from hexbytes import HexBytes
+from ethpm_types import HexBytes
```

Comment:
Just noticed this dependency in here, along with BaseModel (somewhat modified re-import from Pydantic)

QA question, can we reduce this dependency?